tests/system/small/ml/test_llm.py

```diff
@@ -413,9 +413,17 @@ def test_llm_palm_score_params(llm_fine_tune_df_default_index):
     )
 
 
-@pytest.mark.flaky(retries=2)
-def test_llm_gemini_pro_score(llm_fine_tune_df_default_index):
-    model = llm.GeminiTextGenerator(model_name="gemini-pro")
+# test score() function for "gemini-pro" and "gemini-1.5" model
```

Comment:
nit: test name already indicates it. The comment seems redundant.

Reply:
done